### ISSUE TITLE
Error installing conduit via uberenv

### ISSUE THREAD

**Issue by dyokelson** [LLNL/conduit#1100]
Hello, I tried the quick start for conduit on a few different machines and ran the following steps:
```
git clone --recursive https://github.com/llnl/conduit.git
cd conduit
python3 scripts/uberenv/uberenv.py --install --prefix="build"
```
One machine is skylake-gold with gcc/10.2, the other is broadwell with gcc/8.5. I have cmake/3.26 in my path. 
On both I got this same error posted below during the install conduit-develop phase. Please let me know if you have any ideas for a fix or if you need additional information. Thanks!

```
==> Installing conduit-develop-7f52qjlnxehehsvgxkzkroi5jl6hczoh
==> No binary for conduit-develop-7f52qjlnxehehsvgxkzkroi5jl6hczoh found: installing from source
==> No patches needed for conduit
==> conduit: Executing phase: 'hostconfig'
==> conduit: Executing phase: 'cmake'
==> conduit: Executing phase: 'build'
==> Error: FileNotFoundError: [Errno 2] No such file or directory: '/storage/users/dewiy/conduit/build/spack/opt/spack/linux-rhel8-broadwell/gcc-8.5.0/conduit-develop-7f52qjlnxehehsvgxkzkroi5jl6hczoh/.spack/test/cmake_bin_path.txt'

/storage/users/dewiy/conduit/build/spack/var/spack/repos/builtin/packages/conduit/package.py:279, in cmake_bin:
        276        """(Hack) Set/get cmake dependency path."""
        277        filepath = join_path(self.install_test_root, "cmake_bin_path.txt")
        278        if set:
  >>    279            with open(filepath, "w") as out_file:
        280                cmake_bin = join_path(self.spec["cmake"].prefix.bin, "cmake")
        281                out_file.write("{0}\n".format(cmake_bin))
        282        else:

See build log for details:
  /tmp/dewiy/spack-stage/spack-stage-conduit-develop-7f52qjlnxehehsvgxkzkroi5jl6hczoh/spack-build-out.txt

[ERROR: failure of spack install/dev-build]
```

**Comment by dyokelson** [comment]
Also, this directory does not exist '/storage/users/dewiy/conduit/build/spack/opt/spack/linux-rhel8-broadwell/gcc-8.5.0/conduit-develop-7f52qjlnxehehsvgxkzkroi5jl6hczoh/'

**Comment by cyrush** [comment]
@dyokelson thanks for the report, working on seeing if I can reproduce then resolve.

**Comment by cyrush** [comment]
@dyokelson FYI:  I recreated your issue and I resolved the underlying cause in #1108. I also added a CI test for this specific case.
